catalog/resource_sql_table.go

```diff
@@ -242,6 +255,39 @@ func (ti *SqlTableInfo) serializeColumnInfos() string {
 	return strings.Join(columnFragments[:], ", ") // id INT NOT NULL, name STRING, age INT
 }
 
+func serializePrimaryKeyConstraint(constraint ConstraintInfo) string {
```

Comment:
Should we make this as part of the ConstraintInfo struct that we do for getWrappedConstraintName as it will only be used for each constraint info and internally uses ConstraintInfo. getWrappedConstraintName, getWrappedKeyColumnNames and Rely? 

We do the same for SqlTableInfo. serializeColumnInfos, serializeColumnInfo